typescript compile error TS1479

**Describe the bug**
When I try to import eta in my project with

```
import * as Eta from 'eta'
```

I get

```
error TS1479: The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("eta")' call instead.
  To convert this file to an ECMAScript module, change its file extension to '.mts', or add the field `"type": "module"` to 'package.json'.
```

**To Reproduce**

tsconfig.json:
```
{
  "compilerOptions": {
    "allowUnreachableCode": false,
    "noUnusedLocals": true,
    "isolatedModules": true,
    "esModuleInterop": true,
    "noUnusedParameters": false,
    "importHelpers": true,
    "target": "es2017",
    "disableSizeLimit": true,
    "module": "commonjs",
    "noImplicitAny": false,
    "removeComments": false,
    "preserveConstEnums": true,
    "sourceMap": false,
    "downlevelIteration": true,
    "resolveJsonModule": true,
    "moduleResolution": "node16",
    "skipLibCheck": true,
    "lib": [ "es2017", "dom", "dom.iterable", "webworker" ]
  },
  "include": [ "index.ts" ],
  "exclude": [ "node_modules" ]
}
```

index.ts:
```
import * as Eta from 'eta'
console.log(Eta)
```

package.json:
```
{
  "name": "tt",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "tsc --noemit"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "eta": "^1.13.0",
    "tslib": "^2.4.1",
    "typescript": "^4.9.4"
  }
}
```

type: `npx tsc --noemit`

**Expected behavior**

No error :)

**Package & Environment Details**
 - Environment: Node
 - Version: 18.9.0

--- a/package.json
+++ b/package.json
@@ -18,10 +18,10 @@
   "umd:main": "./dist/eta.umd.js",
   "unpkg": "./dist/browser.min.umd.js",
   "module": "./dist/eta.module.mjs",
-  "types": "./dist/types/eta.d.ts",
+  "types": "./dist/types/index.d.ts",
   "source": "src/index.ts",
   "exports": {
-    "types": "./dist/types/eta.d.ts",
+    "types": "./dist/types/index.d.ts",
     "browser": "./dist/browser.min.umd.js",
     "require": "./dist/eta.umd.js",
     "import": "./dist/eta.module.mjs",

